packages/google_maps_flutter/google_maps_flutter/lib/src/google_map.dart

```diff
@@ -135,6 +136,9 @@ class GoogleMap extends StatefulWidget {
   /// The initial position of the map's camera.
   final CameraPosition initialCameraPosition;
 
+  /// True if the map should cache markers when displaying the map.
```

Comment:
We could also give some info on possible issues related to doing this cache, and also when this may indeed result on some performance improvement and when it won't chance much (if that's the case)
```suggestion
  /// If `true`, will cache markers when displaying the map for improved performance.
```